erpnext/accounts/report/financial_statements.html

```diff
@@ -23,6 +23,7 @@
 {% } %}
 <h2 class="text-center">{%= __(report.report_name) %}</h2>
 <h3 class="text-center">{%= filters.company %}</h3>
+<h3 class="text-center">{%= filters.cost_center %}</h3>
```

Comment:
Add this only if filtered by cost center.

Reply:
ok